package.json

```diff
@@ -87,6 +88,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/chuck-adaptive/ui-kit-2022.git"
-  }
+    "url": "git+https://github.com/chuck-adaptive/ui-kit-2022.git"
+  },
+  "bugs": {
+    "url": "https://github.com/chuck-adaptive/ui-kit-2022/issues"
+  },
+  "readme": "ERROR: No README data found!",
+  "homepage": "https://github.com/chuck-adaptive/ui-kit-2022#readme",
+  "_id": "ui-kit-2022@0.0.0"
```

Comment:
Curious about `_id` and `readme`.  Were these autogenerated?   Should `readme` point to the project readme?

Reply:
It's generated. I assume `_id` is used by chromatic, but I have no idea what is supposed to go in the readme field. I can't find any docs on it being standard in `package.json`, so it's probably used by chromatic, too. I don't really know what value expects or what it's for though since I can't find any documentation.

Reply:
Got it.. yeah maybe if its getting auto-generated by chromatic then just let it be.  `homepage` is intriguing me as well be cause I thought that is reserved for noting the relative path that the project is hosted at (for build purposes, if needed to generate a path from the origin), perhaps what's here is serving that purpose though )`chuck-adaptive/ui-kit-2022`.   We can let it be unless it causes future issues.

Reply:
the package.json [docs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#homepage) are very descriptive about what the homepage is for: `The url to the project homepage.` I think it's only purpose is for documenting a project webpage as a resource for package users.